.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "python.pythonPath": "/Users/jatinkatyal/Library/Caches/pypoetry/virtualenvs/ide-core-gw8-j-eW-py3.9/bin/python"
+    "python.pythonPath": "C:\\Users\\Tarun Luthra\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\ide-core-TtH37sq1-py3.8\\Scripts\\python.exe"
```

Comment:
add .vscode to .gitignore 

also remove current .vscode by running `git rm --cached .vscode/`